src/styles/blocks/sending-form.scss

```diff
@@ -0,0 +1,45 @@
+.sending-form {
+  &__input {
+    box-sizing: border-box;
+    padding: 14px 16px;
+    width: 100%;
+    height: 48px;
+    font-family: inherit;
+    font-size: 14px;
+    line-height: 20px;
+    color: $form-text-color;
+    background-color: $form-background-color;
+    border: 1px solid $form-background-color;
+    border-radius: 40px;
+    margin-bottom: 16px;
+    outline: none;
+
+    &--text-area {
+      height: 148px;
+      border-radius: 16px;
+      margin-bottom: 32px;
+    }
+
+    &--submit {
+      padding: 0;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 27px;
+      color: $menu-text-color;
+      cursor: pointer;
+      @include buttonState;
+    }
+
+    &:hover {
+      border: 1px solid $form-border-interactive;
```

Comment:
```suggestion
      border: 1px solid $form-border-interactive;
      
```

Reply:
What do you mean by these changes? It looks like nothing has changed here.